DTXManiaプロジェクト/コード/ステージ/05.選曲/CActSelectPresound.cs

```diff
@@ -85,7 +85,7 @@ public override int On進行描画()
 				if( ( this.ctBGMフェードイン用 != null ) && this.ctBGMフェードイン用.b進行中 )
 				{
 					this.ctBGMフェードイン用.t進行();
-					CDTXMania.Skin.bgm選曲画面.n音量_現在のサウンド = this.ctBGMフェードイン用.n現在の値;
+					CDTXMania.Skin.bgm選曲画面.nAutomationLevel_現在のサウンド = this.ctBGMフェードイン用.n現在の値;
```

Comment:
Renaming for clarity here, as this specific part of the sound level control updates is dedicate to things like fader automation for fade-in and fade-out of things like song selection screen background music.